src/services/recim.ts

```diff
@@ -3,28 +3,71 @@ import http from './http';
 // all Gordon Rec-IM http requests
```

Comment:
Would it make sense to split this file up into a folder containing one service per API resource? e.g. instead of `services/recim.ts`, you'd have `services/recim/activity.ts`, `services/recim/team.ts`, etc.?